doc/classes/ProjectSettings.xml

```diff
@@ -2464,10 +2464,10 @@
 			Number of blur passes to use when computing screen-space indirect lighting. A higher number will result in a smoother look, but will be slower to compute and will have less high-frequency detail.
 		</member>
 		<member name="rendering/environment/ssil/fadeout_from" type="float" setter="" getter="" default="50.0">
-			Distance at which the screen-space indirect lighting effect starts to fade out. Use this hide screen-space indirect lighting at great distances.
+			Distance at which the screen-space indirect lighting effect starts to fade out. Use this to hide screen-space indirect lighting from far away.
```

Comment:
These were the only ever mentions of "_great distance_" in the entire class reference. The closest thing was "_from a long distance_" but hopefully this is nice, too.